docs/flatcar-linux/packet.md

```diff
@@ -110,6 +110,10 @@ module "controller" {
   # Check yours from the `IPs & Networks` tab from your Packet.net account.
   # If an IP block is not allocated yet, try provisioning an instance from the console in
   # that region. Packet will allocate a public IP CIDR.
+  # Note: the private CIDR range has been observed to change unexpectedly, 
+  # although at the time it was an empty/dormant Packet project and there had been some networking upgrades
+  # be sure to manually check this value again when either reusing an older project 
+# Note: Packet does not guarantee this CIDR to be stable if there are no servers deployed in the project and region
```

Comment:
Sorry I wasn't clear, this is just duplicating the notes and seems more confusing, IMHO. Just the second one seems fine. What do you think?

(also, spacing is wrong)

Reply:
sure, this line is fine